tests/test_render_pass.py

```diff
@@ -0,0 +1,124 @@
+from pyvista import _vtk
+from pyvista.plotting.render_passes import RenderPasses
+
+
+# this ideally would be a fixture, but if it's a fixture renderer collects
+# immediately
```

Comment:
Can you explain this comment? Why can't a fixture return these two? Or what do you mean by "collects"?

Reply:
Added in 69243a9ff453f7d292ce9329026b0ef8c43fa355.